scripts/lib/setup_path.py

```diff
@@ -8,9 +8,11 @@
 
 def setup_path() -> None:
     BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-    venv = os.path.realpath(os.path.join(BASE_DIR, "zulip-py3-venv"))
+    venv = os.path.realpath(os.path.join(BASE_DIR, ".venv"))
     if sys.prefix != venv:
-        sys.path = list(filter(lambda p: "/zulip-py3-venv/" not in p, sys.path))
+        sys.path = list(
+            filter(lambda p: "/zulip-py3-venv/" not in p and "/.venv/" not in p, sys.path)
+        )
```

Comment:
Should we add a comment explaining why `/zulip-py3-venv/` is still in here? I assume it's something related to the upgrade path?